docker/build.sh

```diff
@@ -4,27 +4,34 @@ set -ex
 
 DATE=$(date +%m.%y)
 
-docker build -t electronuserland/builder:base -t "electronuserland/builder:base-$DATE" docker/base
+docker build -t devalexandria/builder:base -t "devalexandria/builder:base-$DATE" docker/base
 
 ## NOTE: Order the latest to oldest versions. The most recent node LTS should be tagged as the latest image
 
+# Node 20
+docker build --build-arg NODE_VERSION=20.9.0 --build-arg IMAGE_VERSION=base-$DATE -t devalexandria/builder:20 -t "devalexandria/builder:20-$DATE" -t devalexandria/builder:latest docker/node
```

Comment:
Does it make sense to use 20.15.1 right away? Because 20.09.0 is relatively old (Oct 24, 2023)
[https://nodejs.org/en/blog/release/v20.15.1](blog)